src/Command/DumpCommand.php

```diff
@@ -202,6 +204,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        if (!empty($dumpSettings['gdpr-expressions-file'])) {
+            $dumpSettings['gdpr-expressions'] = json_decode(file_get_contents($dumpSettings['gdpr-expressions-file']),
```

Comment:
Consider splitting the lines like this

```
            $dumpSettings['gdpr-expressions'] = json_decode(
                file_get_contents($dumpSettings['gdpr-expressions-file']),
                true
            )

```

It will make it easier to read